pkg/apiserver/metrics/metrics.go

```diff
@@ -48,6 +48,8 @@ var (
 		prometheus.SummaryOpts{
 			Name: "apiserver_request_latencies_summary",
 			Help: "Response latency summary in microseconds for each verb and resource.",
+			// Make the sliding window of 1h.
+			MaxAge: time.Hour,
```

Comment:
Similar argument for the rest of this structure.  Why are the two help strings slightly different?  That's an example of the general case I mention above.